sections/footer.liquid

```diff
@@ -76,9 +76,10 @@
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
-                      {%- assign image_size = block.settings.image_width | append: 'x' -%}
+                      {%- assign image_size = block.settings.image_width -%}
+                      {%- assign image_size_2x = block.settings.image_width | times: 2 -%}
```

Comment:
```diff
-{%- assign image_size_2x = block.settings.image_width | times: 2 -%}
+{%- assign image_size_2x = block.settings.image_width | times: 2 | at_most: 5760 -%}
```

I think we also need to add `at_most: 5760` to avoid errors originally mentioned here: https://github.com/Shopify/dawn/pull/955#discussion_r759489837